src/LibraryManager.Vsix/Shared/VsHelpers.cs

```diff
@@ -309,6 +312,7 @@ public static void SatisfyImportsOnce(this object o)
 
                 if (File.Exists(configFilePath))
                 {
+                    Telemetry.TrackUserTask($"ProjectContainsLibMan{project.Name}");
```

Comment:
>{project.Name} [](start = 67, length = 14)

Make this a property instead of part of the event name. Project name should be PII.

Reply:
Might be better to use project GUID instead. 

---
In reply to: [200786690](https://github.com/aspnet/LibraryManager/pull/282#discussion_r200786690) [](ancestors = 200786690)